### PR TITLE
MDOT Address Page: Update Unit Tests

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
This PR focuses on creating and updating unit tests for the address page components.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/2346/components/AddressViewField.jsx [hunk 1]
```diff
@@ -42,25 +67,55 @@ const AddressViewField = ({ formData }) => {
...
+              {isBaseAddressDataValid && (
+                <>
+                  {addLine(street)}
+                  {addLine(street2)}
+                  {addLine(street3)}
+                  {isDomesticAddressValid &&
+                    `${city}, ${state} ${postalString}`}
+                  {isInternationalAddressValid &&
+                    `${city}, ${province} ${internationalPostalCode}`}
+                  <br />
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/components/AddressViewField.jsx:83]: Can we remove this linebreak?

File: src/applications/disability-benefits/2346/components/AddressViewField.jsx [hunk 2]
```diff
@@ -42,25 +67,55 @@ const AddressViewField = ({ formData }) => {
...
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/components/AddressViewField.jsx:104]: And this one

Comment (bradl3yC) [reply src/applications/disability-benefits/2346/components/AddressViewField.jsx:104]: Also, if these code blocks are identical with the exception of the addressType comparison, they're not needed

File: src/applications/disability-benefits/2346/tests/AddressViewField.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    const formData = {
+      ...baseData,
+      street: 's1',
+      street2: 's2',
+      postalCode: '12345',
+      country: 'USA',
+      state: 'state',
+    };
+    const wrapper = shallow(
+      <AddressViewField formData={formData} addressType="permanentAddress" />,
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/tests/AddressViewField.unit.spec.jsx:21]: Why are we passing in a new prop called addressType that doesn't exist in any of our schema?

**Review by bradl3yC — APPROVED** [review]

**Review by bradl3yC — APPROVED** [review]

**Review by cvalarida — APPROVED** [review]
Update to `package.json` looks okay. :+1:

**Pull request merged**